src/SuperDumpService/Program.cs

```diff
@@ -5,7 +5,7 @@ namespace SuperDumpService {
 	public static class Program {
 		public static void Main(string[] args) {
 			var host = new WebHostBuilder()
-				.UseKestrel()
+				.UseKestrel(opt => opt.Limits.MaxRequestBodySize = 8589934592L)  // 8gb
```

Comment:
I've moved the configuration into Program. Now, the maxUploadSizeMB is used to set the max request body size.
Alternatively, we could also set the max request body size to long.MaxValue to get the original behavior. The error screen if we exceed the upload size is pretty ugly (plain "Error Status: 0" page).